src/Dfc.CourseDirectory.Core/DataManagement/FileUploadProcessor.Courses.cs

```diff
@@ -19,6 +19,34 @@ namespace Dfc.CourseDirectory.Core.DataManagement
 {
     public partial class FileUploadProcessor
     {
+        public async Task<bool> DeleteCourseUploadRowForProvider(Guid providerId, int rowNumber)
```

Comment:
Nit: whitespace is a little wonky in this method

Reply:
done